.storybook/preview-head.html

```diff
@@ -9,14 +9,14 @@
   }
 
   /* Custom styling to override Storybook docs global styles */
-  .sbdocs-h1 {
+  .sbdocs h1.css-wzniqs {
```

Comment:
what is `css-wzniqs`?

Reply:
It's a class that Storybook adds to elements within the docs to apply their default styles.